mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeansModel.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.rdd.RDD
 @Since("1.6.0")
 @Experimental
 class BisectingKMeansModel @Since("1.6.0") (
-    @Since("1.6.0") val root: ClusteringTreeNode
+    @Since("1.6.0")private[clustering] val root: ClusteringTreeNode
```

Comment:
@yu-iskw Can you confirm if you meant this to be public?  It may not be necessary since most users will just call predict, computeCost, etc.  What do you think?  It does expose a much larger API.